nebula_ros/src/continental/continental_ars548_ros_wrapper.cpp

```diff
@@ -90,6 +90,8 @@ nebula::Status ContinentalARS548RosWrapper::declare_and_get_sensor_config_params
   config.configuration_sensor_port =
     static_cast<uint16_t>(declare_parameter<int>("configuration_sensor_port", param_read_only()));
   config.use_sensor_time = declare_parameter<bool>("use_sensor_time", param_read_write());
+  config.radar_info_rate_subsample =
+    declare_parameter<int>("radar_info_rate_subsample", param_read_write());
```

Comment:
It it's meant as a `param_read_write`, then it would also have added in `on_parameter_change` below. I think it's also fine as `param_read_only` though.

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/5cce98acffcaf1d37e2df56217d79da21c8a66b2